controlplane/migrations/0112_fluffy_marvel_apes.sql

```diff
@@ -0,0 +1,16 @@
+CREATE TABLE IF NOT EXISTS "namespace_cache_warmer_config" (
+	"id" uuid PRIMARY KEY DEFAULT gen_random_uuid() NOT NULL,
+	"namespace_id" uuid NOT NULL,
+	"max_operations_count" integer NOT NULL,
+	"created_at" timestamp with time zone DEFAULT now() NOT NULL,
+	"updated_at" timestamp with time zone,
+	CONSTRAINT "namespace_cache_warmer_config_namespace_id_unique" UNIQUE("namespace_id")
+);
+--> statement-breakpoint
+DO $$ BEGIN
```

Comment:
Should add a migration to create the config for all the namespaces which have cache warmer enabled